server/settings/publish_plugins.py

```diff
@@ -4,19 +4,138 @@
     BaseSettingsModel,
     SettingsField,
     ensure_unique_names,
+    task_types_enum,
 )
 
 
+class LimitGroupsSubmodel(BaseSettingsModel):
+    _layout = "expanded"
+    name: str = SettingsField(title="Name")
+    value: list[str] = SettingsField(
+        default_factory=list,
+        title="Limit Groups"
+    )
+
+
+class EnvSearchReplaceSubmodel(BaseSettingsModel):
+    _layout = "compact"
+    name: str = SettingsField(title="Name")
+    value: str = SettingsField(title="Value")
+
+
 class CollectAYONServerToFarmJobModel(BaseSettingsModel):
     enabled: bool = SettingsField(False, title="Enabled")
 
 
-class CollectDeadlinePoolsModel(BaseSettingsModel):
-    """Settings Deadline default pools."""
+def extract_jobinfo_overrides_enum():
+    """Enum of fields that could be overridden by artist in Publisher UI"""
+    return [
```

Comment:
Small note (from my point of view as developer), `"value"` should be on left side -> easily readable and `"label"` afterwards.